R/utils.R

```diff
@@ -120,29 +124,82 @@ expose_stan_fns <- function(files, target_dir, ...) {
 #'
 #' # Load the script used to generate these examples
 #' # Optionally source this script to regenerate the example
-#' readLines(enw_example(type = "script"))
-enw_example <- function(type = "nowcast") {
-  type <- match.arg(
-    type,
-    choices = c(
-      "nowcast", "preprocessed_observations", "observations", "script"
-    )
-  )
+#' readLines(enw_example(type = "scripts"))
+enw_example <- function(
+  type = c("nowcast", "preprocessed_observations", "observations", "scripts")
+) {
+  type <- match.arg(type)
 
   if (type %in% c("nowcast", "preprocessed_observations", "observations")) {
-    file <- system.file("extdata", paste0(type, ".rds"), package = "epinowcast")
-  } else if (type %in% "script") {
-    file <- system.file("scripts", "germany_example.R", package = "epinowcast")
+    return(readRDS(
+      system.file("extdata", sprintf("%s.rds", type), package = "epinowcast")
+    ))
+  } else if (type == "scripts") {
+    return(list.files(
```

Comment:
This is a change in behaviour as there are multiple example scripts and only one set of saved output. I think it also breaks the example on line 127.

Reply:
there are multiple example files in `examples`, but only one file in `scripts`. not breaking as long as it stays that way, and no longer cares what that script is named.

Reply:
ah, if that's how this is supposed to behave, then okay - so this should point at `examples/germany_dow.R`?

yeah, bug was that it pointed to something that didn't exist in scripts - but was a germany => generate substitution away.